templates/operations.html

```diff
@@ -816,4 +822,24 @@ <h1 id="time-tactic" class="member-title"></h1>
         $('#resultCmd').html(atob(command));
     }
     //# sourceURL=operations.js
+
+    function advHasRepeatableAbilities(selectedAdversary) {
+        if ($('#qflow-' + selectedAdversary).data('has-repeatable') != null) {
+            $("#queuePlanner option").each(function() {
+                if (!this.hasAttribute('data-allow-repeatable-abilities')) {
+                    this.setAttribute('disabled', '');
+                    this.removeAttribute('selected');
+                }
+             });
+            $('#queuePlanner option[value="batch"]').prop('selected', 'selected');
+            $('#queuePlanner').val('batch');
+             stream('The profile selected has repeatable abilities. Some planners do not support repeatable abilities and have been disabled.');
+        }
+        else {
+            $('#queuePlanner option').each(function() { this.removeAttribute('disabled'); this.removeAttribute('selected') });
+            $('#queuePlanner option[value="atomic"]').prop('selected', 'selected');
+            $('#queuePlanner').val('atomic');
+            stream('Selected planner defaulting to ' + $('#queuePlanner').val() + '.');
```

Comment:
hardcoding in atomic as a planner can have some unintended consequences down the line. Let's find a solution that doesn't need a hardcoded value.